app/src/main/java/com/markkolenbrander/capstonenewsapp/views/NewsArticleDatabase.kt

```diff
@@ -18,19 +16,19 @@ private const val DATABASE_VERSION = 1
 @TypeConverters(DataConverter::class)
 abstract class NewsArticleDatabase : RoomDatabase() {
 
-    companion object{
-        private const val DATABASE_NAME = "NewsArticles"
-
-        fun buildDatabase(context: Context): NewsArticleDatabase {
-            return Room.databaseBuilder(
-                context,
-                NewsArticleDatabase::class.java,
-                DATABASE_NAME,
-            )
-                .allowMainThreadQueries()
-                .build()
-        }
-    }
+//    companion object{
+//        private const val DATABASE_NAME = "NewsArticles"
+//
+//        fun buildDatabase(context: Context): NewsArticleDatabase {
+//            return Room.databaseBuilder(
+//                context,
+//                NewsArticleDatabase::class.java,
+//                DATABASE_NAME,
+//            )
+//                .allowMainThreadQueries()
+//                .build()
+//        }
+//    }
```

Comment:
Minor, but you could leave this here and reference it via your AppModule with `NewsArticleService.buildDatabase(...)`